### PR TITLE
#155965573 fix failing tests, add more tests and add new login redirecting feature. 

### PR DESCRIPTION
**Pull request by mutoromike**
**What does this PR do?**

- Fix failing tests on accounts
- Add more account test features
- Add new tests
- Creates a redirecting feature after login

**How should this be manually tested?**

- Clone remote repo to local machine
- Setup project locally and activate the virtual environment
- Run python manage.py test hc.accounts in the terminal

**What are the relevant pivotal tracker stories?**

**The Accounts tests need fixing
ID: #155965573**

### CODE REVIEW

File: hc/accounts/tests/test_check_token.py [hunk 1]
```diff
@@ -22,7 +29,29 @@ def test_it_redirects(self):
         self.assertEqual(self.profile.token, "")
 
     ### Login and test it redirects already logged in
+    def test_it_redirects_loggedin_users(self):
```
Comment (meshnesh) [inline hc/accounts/tests/test_check_token.py:32]: Nice Implementation of the redirect after successful login, although having a doctoring would be best to explain the function

File: hc/accounts/tests/test_check_token.py [hunk 2]
```diff
@@ -22,7 +29,29 @@ def test_it_redirects(self):
...
     ### Login with a bad token and check that it redirects
+    def test_login_with_bad_token(self):
+        url = reverse('hc-check-token', args=['bob', 'bad-token'])
+        response = self.client.post(url)
+        
+        self.assertRedirects(response, reverse('hc-login'))
 
     ### Any other tests?
+    ### Test if login link is sent after registration
+    def test_login_link_is_sent(self):
```
Comment (meshnesh) [inline hc/accounts/tests/test_check_token.py:51]: nice implementation

**Review by meshnesh — COMMENTED** [review]
Good adding doc strings on the test functions

**Comment by lakivisi-zz** [comment]
@meshnesh Fix conflicts

File: hc/accounts/tests/test_check_token.py (hunk 1 above)
Comment (mutoromike) [reply hc/accounts/tests/test_check_token.py:32]: Great! This will be implemented.

**Pull request merged**